hnn_core/network.py

```diff
@@ -484,8 +545,14 @@ def add_evoked_drive(self, name, *, mu, sigma, numspikes, location,
             a function of the space_constant. If float, applies to all target
             cell types. Use dict to create delay->cell mapping.
         space_constant : float
-            Describes lateral dispersion (from column origin) of synaptic
-            weights and delays within the simulated column
+            Describes lateral dispersion (from the column origin) of synaptic
+            weights and delays within the simulated column. The constant is
+            measured in the units of ``inplane_distance`` of
+            :class:`~hnn_core.Network`. For example, for ``space_constant=3``,
+            the weights and delays are modulated by the factor
+            ``exp(-(x / (3 * inplane_distance)) ** 2)``, where ``x`` is the
```

Comment:
```suggestion
            the weights are modulated by the factor
            ``exp(-(x / (3 * inplane_distance)) ** 2)``, where ``x`` is the
```
delays are modulated by a factor that is the inverse of that value